rust/processor/migrations/2024-04-18-173631_fungible_token/up.sql

```diff
@@ -0,0 +1,12 @@
+-- Your SQL goes here
+ALTER TABLE fungible_asset_metadata
+ADD COLUMN supply_v2 NUMERIC,
+ADD COLUMN maximum_v2 NUMERIC;
+
+ALTER TABLE current_token_datas_v2
+ALTER COLUMN supply DROP NOT NULL,
+ALTER COLUMN decimals DROP NOT NULL;
```

Comment:
why are we dropping? I thought we weren't making backward incompatible changes.

Reply:
before the column is not nullable. this only drops the null part to make it nullable.

Reply:
Oh nvm you're making it nullable. Cool.